ert3/engine/_export.py

```diff
@@ -74,10 +74,26 @@ def _prepare_export_responses(
 
     for record_name in responses:
         for iens in range(ensemble_size):
-            url = f"{records_url}/{record_name}?realization_index={iens}"
-            future = ert.storage.load_record(url, ert.data.RecordType.LIST_FLOAT)
-            record = get_event_loop().run_until_complete(future)
-            outputs[record_name].append(record.data)
+            uri = f"{records_url}/{record_name}"
+            future = ert.storage.get_record_metadata(uri)
+            metadata = get_event_loop().run_until_complete(future)
+            if metadata.get("record_type") == ert.data.RecordType.NUMERICAL_TREE:
+                transmitter = ert.storage.StorageRecordTransmitter(
+                    record_name, records_url, iens=iens
+                )
+                transmitter.set_transmitted(
```

Comment:
This is pretty smelly for a couple of reasons: `set_transmitted` is a storage hack, and we're doubling down on leaking uris from storage. 

Do you see any quick and painless way to refactor this to address these smells (to some degree)?

Reply:
Yes, I think so. But don't fully see why not to avoid using it? Is it the usage of transmitters alone; which of-course when created for an existing record needs to set uri via set_transmitted or just that the storage uri leaks into the export?

Reply:
Both. I think calling the record constructor should suffice in order to get a complete record. But the urls are definitely more smelly, so if that is relatively easy to fix …